Testing/Functional/SmokeTest/SmokeTest001.Tests.ps1

```diff
@@ -0,0 +1,85 @@
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSReviewUnusedParameter', 'M365Environment', Justification = 'False positive as rule does not scan child scopes')]
+[CmdletBinding(DefaultParameterSetName='Manual')]
+param (
+    [Parameter(Mandatory = $true, ParameterSetName = 'Auto')]
+    [ValidateNotNullOrEmpty()]
+    [string]
+    $Thumbprint,
+    [Parameter(Mandatory = $true, ParameterSetName = 'Auto')]
+    [ValidateNotNullOrEmpty()]
+    [string]
+    $Organization,
+    [Parameter(Mandatory = $true,  ParameterSetName = 'Auto')]
+    [ValidateNotNullOrEmpty()]
+    [string]
+    $AppId,
+    [Parameter(ParameterSetName = 'Auto')]
+    [Parameter(ParameterSetName = 'Manual')]
+    [Parameter(Mandatory = $false)]
+    [ValidateNotNullOrEmpty()]
+    [string]
+    $M365Environment = 'gcc'
+)
+
+$ScubaModulePath = Join-Path -Path $PSScriptRoot -ChildPath "../../../PowerShell/ScubaGear/ScubaGear.psd1"
+Import-Module $ScubaModulePath
+
+Describe "Smoke Test: Generate Output" {
+    Context "Invoke Scuba for $Organization" {
+        BeforeAll {
+            if ($PSCmdlet.ParameterSetName -eq 'Manual'){
+                Invoke-SCuBA -ProductNames "*" -M365Environment $M365Environment
+            }
+            else {
+                Invoke-SCuBA -CertificateThumbprint $Thumbprint -AppID $AppId -Organization $Organization -ProductNames "*" -M365Environment $M365Environment
+            }
+            $ReportFolders = Get-ChildItem . -directory -Filter "M365BaselineConformance*" | Sort-Object -Property LastWriteTime -Descending
+            [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', 'OutputFolder',
+            Justification = 'Variable is used in another scope')]
+            $OutputFolder = $ReportFolders[0]
+        }
+        It "Item, <Item>, exists" -ForEach @(
+            @{Item = 'BaselineReports.html'; ItemType = 'Leaf'},
+            @{Item = 'TestResults.json'; ItemType = 'Leaf'},
+            @{Item = 'TestResults.csv'; ItemType = 'Leaf'},
+            @{Item = 'ProviderSettingsExport.json'; ItemType = 'Leaf'},
+            @{Item = 'IndividualReports'; ItemType = 'Container'},
+            @{Item = 'IndividualReports/AADReport.html'; ItemType = 'Leaf'},
+            @{Item = 'IndividualReports/DefenderReport.html'; ItemType = 'Leaf'},
+            @{Item = 'IndividualReports/EXOReport.html'; ItemType = 'Leaf'},
+            @{Item = 'IndividualReports/OneDriveReport.html'; ItemType = 'Leaf'},
+            @{Item = 'IndividualReports/PowerPlatformReport.html'; ItemType = 'Leaf'},
+            @{Item = 'IndividualReports/SharePointReport.html'; ItemType = 'Leaf'},
+            @{Item = 'IndividualReports/TeamsReport.html'; ItemType = 'Leaf'},
+            @{Item = 'IndividualReports/images'; ItemType = 'Container'},
+            @{Item = 'IndividualReports/images/angle-down-solid.svg'; ItemType = 'Leaf'},
```

Comment:
I think this is unnecessary and will be annoying to update if we change the logo or anything trivial like that.  Lets just test the image folder exists.